### PR TITLE
Add to dashboard button

### LINKED ISSUE #3613: Add trigger button to slickGrid table
Add a button on hover to the current indicators table:

imagen

This button must display another modal with the serialized data from the selected row: name, value, objective, date

### PR DESCRIPTION
**Pull request by Crashillo**
Closes #3613 

## :v: What does this PR do?
Attachs a floating button to each table-type custom field row, in order to trigger a modal window.
Modal content is dummy.

### 📷 Screenshots
imagen

### CODE REVIEW

File: app/views/gobierto_admin/gobierto_common/custom_fields/forms/_plugin.html.erb [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+    data-configuration="<%= record.custom_field.configuration.plugin_configuration.to_json %>">
+    <%= label_tag :custom_field_id, record.custom_field.name %>
+    <%= hidden_field_tag "#{ input_base_name }[#{ record.uid }][custom_field_id]", record.custom_field_id %>
+    
+    <div class="custom_field_value">
+      <%= send(record.field_tag, "#{ input_base_name }[#{ record.uid }][value]", record.payload.to_json, record.tag_attributes) %>
+
+      <div class="data-container"></div>
+
+      <a href="/admin/plans/<%= record.custom_field.instance_id %>/dashboards/list" add-dashboard-btn class="open_remote_modal">
```
Comment (ferblape) [inline app/views/gobierto_admin/gobierto_common/custom_fields/forms/_plugin.html.erb]: Hardcoded URl, use a Rails helper

File: app/views/gobierto_admin/gobierto_dashboards/dashboards/list.html.erb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<div class="modal">
+  <h2>Lorem ipsum dolor sit amet consectetur adipisicing elit. Blanditiis, illo.</h2>
```
Comment (ferblape) [inline app/views/gobierto_admin/gobierto_dashboards/dashboards/list.html.erb:2]: Eliminatem latim

File: app/views/gobierto_admin/gobierto_dashboards/dashboards/list.html.erb [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+  <pre><%= params.permit!.to_json %></pre>
```
Comment (ferblape) [inline app/views/gobierto_admin/gobierto_dashboards/dashboards/list.html.erb:3]: What's doing exactly this pre block? Are you reading the params from there or it's a debug that you forgot to remove?

**Review by ferblape — CHANGES_REQUESTED** [review]

File: app/views/gobierto_admin/gobierto_dashboards/dashboards/list.html.erb (hunk 2 above)
Comment (Crashillo) [reply app/views/gobierto_admin/gobierto_dashboards/dashboards/list.html.erb:2]: @entantoencuanto this file should be removed, shouldn't it?

Comment (entantoencuanto) [reply app/views/gobierto_admin/gobierto_dashboards/dashboards/list.html.erb:2]: Yes, I´ll fix it

Comment (Crashillo) [reply app/views/gobierto_admin/gobierto_dashboards/dashboards/list.html.erb:2]: No no wait

Comment (Crashillo) [reply app/views/gobierto_admin/gobierto_dashboards/dashboards/list.html.erb:2]: I was mixing stuff, this is another PR. Do not pay attention to this file @ferblape. I guess this gonna be removed when master is merged

**Comment by Crashillo** [comment]
Tests are passing, we can merge

**Comment by ferblape** [comment]
I added a few comments...

File: app/views/gobierto_admin/gobierto_dashboards/dashboards/list.html.erb (hunk 2 above)
Comment (Crashillo) [reply app/views/gobierto_admin/gobierto_dashboards/dashboards/list.html.erb:2]: Resolve this comments as they're gonna be replaced in the consequent PRs

File: app/views/gobierto_admin/gobierto_dashboards/dashboards/list.html.erb (hunk 3 above)
Comment (Crashillo) [reply app/views/gobierto_admin/gobierto_dashboards/dashboards/list.html.erb:3]: Resolve this comments as they're gonna be replaced in the consequent PRs

**Comment by Crashillo** [comment]
I did ignore you :trollface: Oooooookay, I'll keep one of them.... 😇

File: app/views/gobierto_admin/gobierto_common/custom_fields/forms/_plugin.html.erb (hunk 1 above)
Comment (entantoencuanto) [reply app/views/gobierto_admin/gobierto_common/custom_fields/forms/_plugin.html.erb]: Fixed

**Comment by entantoencuanto** [comment]
This PR should be closed in favor of #3626, because #3626 is built on top of this (a rebase has been made) and fixes the problems which prevents the merge of this one

**Comment by ferblape** [comment]
closed or merged?

**Comment by ferblape** [comment]
It doesn't work in staging, I'm testing in https://diba-plan.gobify.net/admin/plans/41/projects/11949/edit, it raises a JS error

**Comment by Crashillo** [comment]
It's not clear to me neither. If we close this one, the next review will be more complicate, as it'll fetch all the edited stuff here.

What I mean is, the purpose of split features into atomic PRs is due to it will make easier its revision. If we reject this PRs in favour of the next one, we're not gaining anything against the one-single-PR way

**Comment by furilo** [comment]
Haven't you already been making reviews?

**Comment by Crashillo** [comment]
Review comments belong to one PR. If I ask you to edit something in PR-1, and you do it in PR-2, it becomes more difficult to trace back there what was the original issue, since in PR-2 won't have such comments.

**Comment by entantoencuanto** [comment]
The problem is that if we merge this PR in master the button will open a window which do nothing with lorem ipsum content, this is an incomplete functionality.

**Pull request closed without merging**